data/exceptions.json

```diff
@@ -145,9 +145,18 @@
   {"period": "Lunch", "time": "11:45-12:30"}
 ]},
 "NoSchool":{
-  "MS": [],
-
-  "US": []
+  "imageProperties": {
```

Comment:
maybe call this allDayEvent, to be more generic

with keys
'name'
'imagePrefix'
'numImages' (I checked the style guide and camel-case is recommended for JSON files)